settings.php

```diff
@@ -43,6 +43,11 @@
         get_string('settings:chromiumpath_desc', 'tool_pdfpages'),
         ''));
 
+    $settings->add(new admin_setting_configtext('tool_pdfpages/responsetimeout',
+        get_string('settings:responsetimeout', 'tool_pdfpages'),
+        get_string('settings:responsetimeout_desc', 'tool_pdfpages'),
+        2, PARAM_INT));
```

Comment:
Why is it 2 by default?

Reply:
This is to make sure that the patch won't affect default behaviour.
2 sec is a default value in `chrome-php`: https://github.com/chrome-php/chrome/blob/f3deb4b31606309680bd587ea306499e462d0caa/src/Communication/ResponseReader.php#L112-L113

Reply:
If it fails, should it be longer by default?

Reply:
Does 10 sec sound good to you?

Reply:
```suggestion
        10, PARAM_INT));
```

Reply:
if that fixes our issue, then sure.